platform/linux-generic/pktio/pcap.c

```diff
@@ -163,7 +165,8 @@ static int pcapif_init(odp_pktio_t id ODP_UNUSED, pktio_entry_t *pktio_entry,
 
 static int pcapif_close(pktio_entry_t *pktio_entry)
 {
-	pkt_pcap_t *pcap = &pktio_entry->s.pkt_pcap;
+	pktio_ops_pcap_data_t *pcap =
+		&pktio_entry->ops_data(pcap);
```

Comment:
declaration fins in one line (there are others in this file too)

Reply:
Run out of 80 characters?